Issue #KN-133 fix: Cloud service provider storage header changes

vaibhavbhuva opened the pull request.

vaibhavbhuva committed:
Issue #KN-133 fix: Cloud service provider storage header changes

vinukumar-vs commented on app/scripts/contenteditor/ce-config.js:
```diff
@@ -18,7 +18,25 @@ org.ekstep.contenteditor.config = {
 	localDispatcherEndpoint: '/app/telemetry',
-	previewURL: '/content/preview/preview.html'
+	previewURL: '/content/preview/preview.html',
+	cloudStorage: {
```
How this config can be changed by any adopter?

vaibhavbhuva replied:
@vinukumar-vs We are not changing the way of passing the configuration to the editor. Just added new property to the existing configuration is still remain the same as is it.

vaibhavbhuva replied:
**For example:** 
```
window.context = {
  ....
  presigned_headers: {
      'x-ms-blob-type': 'BlockBlob'
  }
  ....         
}
```

vaibhavbhuva committed:
Issue #KN-133 fix: Replaced package gulp-uglify  to gulp terser

Link: https://stackoverflow.com/questions/38886840/how-to-solve-this-minification-error-on-gulp

vaibhavbhuva committed:
Issue #KN-133 fix: Removed default values

vaibhavbhuva committed:
Issue #KN-133 fix: Test cases issue fixes

vaibhavbhuva committed:
Issue #KN-133 fix: SonarQube issue fixes

vaibhavbhuva committed:
Issue #KN-133 fix: Test cases issue fixes

vaibhavbhuva committed:
Issue #KN-133 fix: Test cases issue fixes

vaibhavbhuva committed:
Issue #KN-133 fix: Test cases issue fixes

vaibhavbhuva committed:
Issue #KN-133 fix: Test cases issue fixes

vaibhavbhuva committed:
Issue #KN-133 fix: Test cases issue fixes

vaibhavbhuva committed:
Issue #KN-133 fix: Test cases issue fixes

vaibhavbhuva committed:
Issue #KN-133 fix: Test cases issue fixes

vaibhavbhuva committed:
Issue #KN-133 fix: Test cases issue fixes

vaibhavbhuva committed:
Issue #KN-133 fix: Test cases issue fixes

vaibhavbhuva committed:
Issue #KN-133 fix: Test cases issue fixes

vaibhavbhuva committed:
Issue #KN-133 fix: Test cases issue fixes

vaibhavbhuva committed:
Issue #KN-133 fix: Test cases issue fixes
